packages/experimental/src/components/ExampleComponent/_example-component.scss

```diff
@@ -21,11 +22,11 @@
     --#{$pkg-prefix}-border-color: transparent;
   }
 
-  .#{$block-class}--boxed-set {
+  .#{$block-class}.#{$block-class}--boxed-set {
```

Comment:
Ooops.

```suggestion
  .#{$block-class}--boxed-set {
```

Reply:
This was your suggestion, in #441 option A, that all component SCSS should include block class in selector.